crates/experimentation_client/src/interface.rs

```diff
@@ -211,17 +211,20 @@ pub extern "C" fn expt_get_satisfied_experiments(
         Some(prefix_list)
     };
 
-    let local = task::LocalSet::new();
-    let experiments = local.block_on(&Runtime::new().unwrap(), unsafe {
-        (*client).get_satisfied_experiments(&context, prefix_list)
-    });
-    let experiments = unwrap_safe!(
-        serde_json::to_value(experiments),
-        return std::ptr::null_mut()
-    );
-    serde_json::to_string(&experiments)
-        .map(|exp| rstring_to_cstring(exp).into_raw())
-        .unwrap_or_else(|err| error_block(err.to_string()))
+    EXP_RUNTIME.block_on(async move {
+        unsafe {
+            unwrap_safe!(
+                (*client)
+                    .get_satisfied_experiments(&context, prefix_list)
+                    .await
+                    .map(|exp| {
+                        rstring_to_cstring(serde_json::to_value(exp).unwrap().to_string())
+                            .into_raw()
+                    }),
+                std::ptr::null_mut()
+            )
+        }
+    })
 }
```

Comment:
@pratikmishra356 we can't use EXP_RUNTIME for this, it needs localSet for this. Can we keep the same code, just unwrap safely and return that